src/experimental/renderer.js

```diff
@@ -5,20 +5,24 @@ import Output from './output';
 import {setStyle} from './dom';
 
 // Since <Static> components can be placed anywhere in the tree, this helper finds and returns them
-const findStaticNode = node => {
+const findStaticNodes = node => {
+	const nodes = [];
+
 	if (node.unstable__static) {
-		return node;
+		nodes.push(node);
 	}
 
 	for (const childNode of node.childNodes) {
 		if (childNode.unstable__static) {
-			return childNode;
+			nodes.push(childNode);
 		}
 
 		if (Array.isArray(childNode.childNodes) && childNode.childNodes.length > 0) {
-			return findStaticNode(childNode);
+			nodes.push(...findStaticNodes(childNode));
```

Comment:
I'm a bit concerned about the impact of this change in terms of performance. Removing the restriction on having just one `<Static>` component means that now we have to traverse entire component tree to find all `<Static>` components.

I think there are few things we can do here:

1. Allow `<Static>` component to only be placed at top-level of your app (can't be nested deep in the tree)
2. Remember references to `<Static>` components from the first time we find them and avoid calling `findStaticNodes` on every render
3. Add `useStatic()` hook which, when called, tells Ink that there's a new `<Static>` component in the tree and sends a reference to it.

However, all of these don't sound like long-term solutions, there's something hacky about each of them. Now that I think of it, we can't actually allow multiple `<Static>` components through CLI app, because there's no way for developer to know in which order they will be written to stdout/stderr. That basically means we should allow only 2 `<Static>` components, one for stdout and one for stderr. Perhaps we could have either `<Static stream="stdout|stderr">` or `<Stdout>` + `<Stderr>` components. 

Sorry for the brain dump! I'll experiment with that idea and let you know ;)

Reply:
Yeah that makes sense. I have another idea. What if static took either a string, or another component that targets which stream?

```
<Static>
  <Log type="stderr">Foo</Log>
</Static>
```